Investigate NATS issues

rbradford opened the pull request.

rbradford committed:
nats: Fetch xenial image from Azure storage

Caching the image improves download performance significantly.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

rbradford committed:
nats: Use QMP quit for exiting in ACPI tables test

This is much quicker than the shutdown method previously used and what
is employed by most tests.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

sboeuf commented on tools/CI/run_nats.sh:
```diff
@@ -54,6 +54,7 @@ if [ ! -f "$WORKLOADS_DIR"/"$CLEAR_KERNEL" ]; then
 fi
 
 if [ ! -f "$WORKLOADS_DIR"/"$UBUNTU_IMAGE" ]; then
+   wget -nv https://nemujenkinsstorage.blob.core.windows.net/images/xenial-server-cloudimg-amd64-uefi1.img ||
```
Nit: from your commit message: s/signifcantly/significantly/

rbradford replied:
Thanks, I fixed that.

rbradford committed:
nats: Temporarily disable CPU hotplug test on virt

This is test has become very flaky and is making it hard to see other
test issues. Temporarily disable this and track progress in a github
issue.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>
